tests/system/python/api/test_statistics.py

```diff
@@ -143,9 +143,8 @@ def test_statistics_history_with_service_enabled(self, start_south_coap, fledge_
         stats = utils.serialize_stats_map(jdoc)
         assert 1 == stats[ASSET_NAME.upper()]
         assert 1 == stats['READINGS']
-        
         # Allow stats collector schedule to run i.e. by default 15s
-        print("Waiting for 15 seconds for FOGL-8738")
+        print("Waiting for 16 seconds for delay caused by FOGL-8738 - north statistics update thread de-prioritised....")
```

Comment:
Do we really need to wait 16 seconds? How are you calculating this number? Are we good on Rpi? All supported engines verified?